gotocompany/compass/v1beta1/service.proto

```diff
@@ -1198,6 +1212,14 @@ message CreateAssetProbeResponse {
   string id = 1;
 }
 
+message SyncAssetsRequest {
+  repeated string index_names = 1;
+}
+
+message SyncAssetsResponse {
+  string job_id = 1;
```

Comment:
do you also want to add a job status check api ? will it be in a separate PR ? @batrov

Reply:
I think we can consider that later, for now we will only throw status accepted after triggering this API